src/main.rs

```diff
@@ -1,99 +1,108 @@
-extern crate hyper;
-extern crate hyper_rustls;
-extern crate yup_oauth2 as oauth2;
+//extern crate datastore_entity;
 extern crate google_datastore1 as datastore1;
 
-extern crate datastore_entity;
-use datastore_entity::DatastoreEntity;
 
+use datastore_entity::{DatastoreEntity, DatastoreProperties, DatastoreManaged};
 
-use datastore1::LookupRequest;
-use datastore1::Key;
-use datastore1::PathElement;
 
-use std::default::Default;
-use datastore1::Datastore;
-use yup_oauth2 as oauth;
+use datastore1::Client;
+use datastore1::schemas::{Key, PathElement, LookupRequest, LookupResponse};
```

Comment:
`use google_datastore1::...................` and just remove `extern crate` above completely instead of my old comment